pom.xml

```diff
@@ -401,7 +401,6 @@
                                         <source>${project.basedir}/src/main/311+-all/scala</source>
                                         <source>${project.basedir}/src/main/311until320-noncdh/scala</source>
                                         <source>${project.basedir}/src/main/31xdb/scala</source>
-                                        <source>${project.basedir}/src/main/312db/scala</source>
```

Comment:
#4257 was just committed and added a file under 312db, and I just realized this PR proposes to remove that directory.  The new AQEUtils file will need to be renamed to 31xdb to avoid breaking the Databricks 9.1 build.

Reply:
OK, let's only retain 31xdb.